frontend/src/modules/member/components/bulk/bulk-edit-attribute-popover.vue

```diff
@@ -0,0 +1,382 @@
+<template>
+  <app-dialog v-if="computedVisible" v-model="computedVisible" title="Edit attribute">
```

Comment:
Same here in the title?

Reply:
Same goes here as well, as mentioned in the previous comment!